pkgs/by-name/pa/package-version-server/package.nix

```diff
@@ -0,0 +1,42 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  openssl,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "package-version-server";
+  version = "0.0.4";
+
+  src = fetchFromGitHub {
+    owner = "zed-industries";
+    repo = "package-version-server";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-Ht5huLugAIj2FTMywdvtxjeM30klwwV8mdq36AxJ2IY=";
+  };
+
+  cargoHash = "sha256-AJldLcUTxenqVFHFPPMaga7QjnjAtyAdbj6OO1DB9IU=";
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    openssl
+  ];
+
+  passthru = {
+    updateScript = nix-update-script { };
+  };
+
+  meta = {
+    description = "A language server that handles hover information in package.json files";
```

Comment:
```suggestion
    description = "Language server that handles hover information in package.json files";
```

Reply:
Yeah, I've read the guidelines.  I'll make the change, since it's enforced, apparently.

Do you have a post or something that explains the reasoning behind this? This is a very weird convention, and I don't think it's a good one.

Reply:
There was some discussion in #317959